src/main/java/pl/coderslab/web/AddNewRecipe.java

```diff
@@ -0,0 +1,53 @@
+package pl.coderslab.web;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import pl.coderslab.dao.AdminDao;
+import pl.coderslab.dao.RecipeDao;
+import pl.coderslab.model.Recipe;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+@WebServlet("/add")
```

Comment:
powinna być chyba ścieżka `/app/recipe/add` - ale nie wiem czy specjalnie tak zostawiłeś, więc zostawiam komentarz

Reply:
zapomniałem zmienić - wrzuciłem commit z poprawką.

Reply:
Wygląda ładnie, przetestuję na moim test branchu i jak będzie ok to potwierdzę.